src/app/services/components/ServiceInsightDetails.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div class="stack">
+    <KCard>
+      <template #body>
+        <div
+          class="columns"
+          style="--columns: 4;"
+        >
```

Comment:
On a 15" screen one of the column moves to the second row:

<img width="1420" alt="image" src="https://github.com/kumahq/kuma-gui/assets/702355/9edd9caf-c7f5-4731-9e92-cc2057e80039">

It's strange as it looks like there's a lot of real estate to put everything on the same row. After more digging it seems like the sidebar on Kuma is 280px whereas it's 240px on Konnect. I'm going to submit a PR to fix this as it also seems to solve this line break.

Reply:
https://github.com/kumahq/kuma-gui/pull/1354